### ISSUE TITLE
cannot expose AZURACAST_BASE_URL to azurarelay

### ISSUE THREAD

**Issue by marekdziem** [AzuraCast/AzuraCast#5793]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

 Rolling Release #39c0372 (2022-10-09 2:24)

### What happened?

hi there - I'm posting this as an issue here hoping to get some insight into whether the issue exists stems from the parent azuracast  service, or a bug in azurarelay setup.

see https://github.com/AzuraCast/AzuraRelay/issues/47

Hi Marek here from [www.eclipse-streaming.co.za](http://www.eclipse-streaming.co.za/)

calling #BusterNeece and #Vaalyn !! (via Maak :) )

What a great project!
I have run into an issue trying to install Azura relay ;)

much frustration....why does a legitimate azuracast URL not function!?
It's a digital oceans droplet container and I've tried the IP address as well!

https://eclipse-radio.eclipse-streaming.co.za/api/nowplaying returns happy JSON for parsing when opened directly/

no ways I had to wait 3000ms for the installer to break - its instant.
please help!

AzuraRelay Setup
Welcome to AzuraRelay! Provide the following items to finish setup.
AzuraCast Installation URL
AzuraRelay has to connect to a "parent" AzuraCast instance to relay its broadcast(s).
Provide the base URL of that installation (including "http://" or "https://") to continue.

AzuraCast Installation URL []:

https://eclipse-radio.eclipse-streaming.co.za/

```
[2022-10-16T23:13:42.283720+00:00] AzuraRelay.ERROR: HTTP client GET call to https://eclipse-radio.eclipse-streaming.co.za/api/nowplaying produced response NULL [] []

[ERROR] Could not connect to AzuraCast instance at https://eclipse-radio.eclipse-streaming.co.za/: cURL error
28: Connection timed out after 3000 milliseconds (see
https://curl.haxx.se/libcurl/c/libcurl-errors.html) for
https://eclipse-radio.eclipse-streaming.co.za/api/nowplaying
/var/azurarelay/www/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php L210: #0
/var/azurarelay/www/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php(158):
GuzzleHttp\Handler\CurlFactory::createRejection()
https://github.com/AzuraCast/AzuraRelay/issues/1 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php(110):
GuzzleHttp\Handler\CurlFactory::finishError()
https://github.com/AzuraCast/AzuraRelay/issues/2 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/Handler/CurlHandler.php(47):
GuzzleHttp\Handler\CurlFactory::finish()
https://github.com/AzuraCast/AzuraRelay/issues/3 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/Handler/Proxy.php(28):
GuzzleHttp\Handler\CurlHandler->__invoke()
https://github.com/AzuraCast/AzuraRelay/issues/4 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/Handler/Proxy.php(48):
GuzzleHttp\Handler\Proxy::GuzzleHttp\Handler{closure}()
https://github.com/AzuraCast/AzuraRelay/issues/5 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/Middleware.php(201):
GuzzleHttp\Handler\Proxy::GuzzleHttp\Handler{closure}()
https://github.com/AzuraCast/AzuraRelay/issues/6 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/PrepareBodyMiddleware.php(35):
GuzzleHttp\Middleware::GuzzleHttp{closure}()
https://github.com/AzuraCast/AzuraRelay/issues/7 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/Middleware.php(31):
GuzzleHttp\PrepareBodyMiddleware->__invoke()
https://github.com/AzuraCast/AzuraRelay/issues/8 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/RedirectMiddleware.php(71):
GuzzleHttp\Middleware::GuzzleHttp{closure}()
https://github.com/AzuraCast/AzuraRelay/pull/9 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/Middleware.php(61):
GuzzleHttp\RedirectMiddleware->__invoke()
https://github.com/AzuraCast/AzuraRelay/issues/10 /var/azurarelay/www_tmp/CompiledContainer.php(26): GuzzleHttp\Middleware::GuzzleHttp{closure}()
https://github.com/AzuraCast/AzuraRelay/issues/11 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/HandlerStack.php(75):
CompiledContainer::{closure}()
https://github.com/AzuraCast/AzuraRelay/issues/12 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/Client.php(331):
GuzzleHttp\HandlerStack->__invoke()
https://github.com/AzuraCast/AzuraRelay/pull/13 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/Client.php(168): GuzzleHttp\Client->transfer()
https://github.com/AzuraCast/AzuraRelay/issues/14 /var/azurarelay/www/vendor/guzzlehttp/guzzle/src/Client.php(187):
GuzzleHttp\Client->requestAsync()
https://github.com/AzuraCast/AzuraRelay/issues/15 /var/azurarelay/www/vendor/azuracast/php-api-client/src/AbstractClient.php(54):
GuzzleHttp\Client->request()
https://github.com/AzuraCast/AzuraRelay/issues/16 /var/azurarelay/www/vendor/azuracast/php-api-client/src/Client.php(24):
AzuraCast\Api\AbstractClient->request()
https://github.com/AzuraCast/AzuraRelay/issues/17 /var/azurarelay/www/src/Console/Command/SetupCommand.php(48): AzuraCast\Api\Client->nowPlaying()
https://github.com/AzuraCast/AzuraRelay/pull/18 /var/azurarelay/www/vendor/symfony/console/Helper/QuestionHelper.php(467):
App\Console\Command\SetupCommand->App\Console\Command{closure}()
https://github.com/AzuraCast/AzuraRelay/issues/19 /var/azurarelay/www/vendor/symfony/console/Helper/QuestionHelper.php(74):
Symfony\Component\Console\Helper\QuestionHelper->validateAttempts()
https://github.com/AzuraCast/AzuraRelay/pull/20 /var/azurarelay/www/vendor/symfony/console/Style/SymfonyStyle.php(348):
Symfony\Component\Console\Helper\QuestionHelper->ask()
https://github.com/AzuraCast/AzuraRelay/pull/21 /var/azurarelay/www/src/Console/Command/SetupCommand.php(62):
Symfony\Component\Console\Style\SymfonyStyle->askQuestion()
https://github.com/AzuraCast/AzuraRelay/pull/22 /var/azurarelay/www/vendor/symfony/console/Command/Command.php(291):
App\Console\Command\SetupCommand->execute()
https://github.com/AzuraCast/AzuraRelay/pull/23 /var/azurarelay/www/vendor/symfony/console/Application.php(1007):
Symfony\Component\Console\Command\Command->run()
https://github.com/AzuraCast/AzuraRelay/pull/24 /var/azurarelay/www/vendor/symfony/console/Application.php(299):
Symfony\Component\Console\Application->doRunCommand()
https://github.com/AzuraCast/AzuraRelay/pull/25 /var/azurarelay/www/vendor/symfony/console/Application.php(171):
Symfony\Component\Console\Application->doRun()
https://github.com/AzuraCast/AzuraRelay/pull/26 /var/azurarelay/www/bin/console(11): Symfony\Component\Console\Application->run()
https://github.com/AzuraCast/AzuraRelay/issues/27 {main}
```

**Comment by maaakmal** [comment]
Hi Folks
No luck so far with trying to get this working
Any help or suggestions much appreciated
Thanks Mark

**Comment by BusterNeece** [comment]
@marekdziem I'm wondering if your DNS is set up correctly on that server. A lot of the time, cURL resolution errors are caused by DNS problems...it may be worth investigating that.

**Comment by marekdziem** [comment]
HI @BusterNeece I wondered too so tried the direct ip address and still no joy.
https://165.227.140.167/api/nowplaying
also tried another host running azuracast and still the same issue.
is there a way to set the global system curl timeout?

**Comment by BusterNeece** [comment]
@marekdziem I greatly extended the timeout for offsite API calls being made by AzuraRelay. You can update your installation using the main update command or just running `docker-compose pull`, then re-running the installer, and hopefully it should work without issue now.

**Comment by marekdziem** [comment]
@BusterNeece ...ah no joy either!!

can we confirm the exact format to input:
https://eclipse-radio.eclipse-streaming.co.za
https://eclipse-radio.eclipse-streaming.co.za/
http://eclipse-radio.eclipse-streaming.co.za
http://eclipse-radio.eclipse-streaming.co.za/
https://165.227.140.167/
http://165.227.140.167/
https://165.227.140.167
http://165.227.140.167

or does it expect a trailing path /api/nowplaying ?

**Comment by marekdziem** [comment]
root@eclipserelay:/var/azurarelay# curl https://eclipse-radio.eclipse-streaming.co.za/api/nowplaying  
returns:

[{"station":{"id":2,"name":"Midlands FM","shortcode":"midlands_fm","description":"Midlands FM","frontend":"icecast","backend":"liquidsoap","listen_url":"https://eclipse-radio.eclipse-streaming.co.za/listen/midlands_fm/radio.mp3","url":"https://mfm.mymidlands.co.za/","public_player_url":"https://eclipse-radio.eclipse-streaming.co.za/public/midlands_fm","playlist_pls_url":"https://eclipse-radio.eclipse-streaming.co.za/public/midlands_fm/playlist.pls","playlist_m3u_url":"https://eclipse-radio.eclipse-streaming.co.za/public/midlands_fm/playlist.m3u","is_public":true,"mounts":[{"id":2,"name":"/radio.mp3 (128kbps MP3)","url":"https://eclipse-radio.eclipse-streaming.co.za/listen/midlands_fm/radio.mp3","bitrate":128,"format":"mp3","listeners":{"total":0,"unique":0,"current":0},"path":"/radio.mp3","is_default":true}],"remotes":[{"id":7,"name":"Midlands Radio CDN","url":"http://iceant.antfarm.co.za/midlandsfm.mp3","bitrate":null,"format":null,"listeners":{"total":0,"unique":0,"current":0}}],"hls_enabled":true,"hls_url":"https://eclipse-radio.eclipse-streaming.co.za/hls/midlands_fm/live.m3u8","hls_listeners":1},"listeners":{"total":1,"unique":1,"current":1},"live":{"is_live":true,"streamer_name":"Classic Mix","broadcast_start":1666301208,"art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/2/streamer/2%7C1665760029/art"},"now_playing":{"sh_id":76105,"played_at":1666347985,"duration":0,"playlist":"","streamer":"Classic Mix","is_request":false,"song":{"id":"6036f7651919b299fd611d2c28437d00","text":" - Live Broadcast","artist":"","title":"Live Broadcast","album":"","genre":"","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/2/streamer/2%7C1665760029/art","custom_fields":[]},"elapsed":478,"remaining":0},"playing_next":{"cued_at":1666330730,"played_at":1666348454,"duration":124,"playlist":"default","is_request":false,"song":{"id":"9a5d1d091a3ea25909196ae2732d8ad2","text":"the smiths - girlfriend in a coma","artist":"the smiths","title":"girlfriend in a coma","album":"","genre":"","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/2/art/67849f33e52d69830b586c0c-1661865073.jpg","custom_fields":[]}},"song_history":[{"sh_id":76104,"played_at":1666347978,"duration":569,"playlist":"default","streamer":"Classic Mix","is_request":false,"song":{"id":"af7e1016635ef41c8a8bc2188d83b32c","text":"Pink Floyd - Comfortably Numb","artist":"Pink Floyd","title":"Comfortably Numb","album":"Pulse","genre":"Rock","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/2/art/d6eb0413313737108ca9a0fa-1661864986.jpg","custom_fields":[]}},{"sh_id":75947,"played_at":1666330737,"duration":17241,"playlist":"","streamer":"Classic Mix","is_request":false,"song":{"id":"6036f7651919b299fd611d2c28437d00","text":" - Live Broadcast","artist":"","title":"Live Broadcast","album":"","genre":"","isrc":"","lyrics":"","art":"https://mfm.mymidlands.co.za/wp-content/uploads/2022/07/1.png","custom_fields":[]}},{"sh_id":75943,"played_at":1666330504,"duration":233,"playlist":"default","streamer":"Classic Mix","is_request":false,"song":{"id":"a54f8d7ec4d727f647d9c681e3639185","text":"Bruno Mars - Locked Out Of Heaven","artist":"Bruno Mars","title":"Locked Out Of Heaven","album":"","genre":"","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/2/art/8cf7464ff6f3767e78b05119-1661864809.jpg","custom_fields":[]}},{"sh_id":75788,"played_at":1666313848,"duration":16656,"playlist":"","streamer":"Classic Mix","is_request":false,"song":{"id":"6036f7651919b299fd611d2c28437d00","text":" - Live Broadcast","artist":"","title":"Live Broadcast","album":"","genre":"","isrc":"","lyrics":"","art":"https://mfm.mymidlands.co.za/wp-content/uploads/2022/07/1.png","custom_fields":[]}},{"sh_id":75787,"played_at":1666313841,"duration":212,"playlist":"default","streamer":"Classic Mix","is_request":false,"song":{"id":"55724806fb3506a774a77a54383a2d71","text":"Live - Sun","artist":"Live","title":"Sun","album":"The Distance to Here","genre":"Rock","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/2/art/6a6162b3d049d095ca91dbb1-1661864959.jpg","custom_fields":[]}}],"is_online":true,"cache":null},{"station":{"id":4,"name":"Plaas Radio","shortcode":"plaas_radio","description":"Streaming groovy tunes to the folks at Casa de Bonos","frontend":"icecast","backend":"liquidsoap","listen_url":"https://eclipse-radio.eclipse-streaming.co.za/listen/plaas_radio/radio.mp3","url":"","public_player_url":"https://eclipse-radio.eclipse-streaming.co.za/public/plaas_radio","playlist_pls_url":"https://eclipse-radio.eclipse-streaming.co.za/public/plaas_radio/playlist.pls","playlist_m3u_url":"https://eclipse-radio.eclipse-streaming.co.za/public/plaas_radio/playlist.m3u","is_public":true,"mounts":[{"id":4,"name":"/radio.mp3 (128kbps MP3)","url":"https://eclipse-radio.eclipse-streaming.co.za/listen/plaas_radio/radio.mp3","bitrate":128,"format":"mp3","listeners":{"total":0,"unique":0,"current":0},"path":"/radio.mp3","is_default":true}],"remotes":[],"hls_enabled":true,"hls_url":"https://eclipse-radio.eclipse-streaming.co.za/hls/plaas_radio/live.m3u8","hls_listeners":0},"listeners":{"total":0,"unique":0,"current":0},"live":{"is_live":false,"streamer_name":"","broadcast_start":null,"art":null},"now_playing":{"sh_id":76106,"played_at":1666348033,"duration":517,"playlist":"TraktaTrax","streamer":"","is_request":false,"song":{"id":"4fabe7cbb80f63b4bff7eee152e9c547","text":"Led Zeppelin - Kashmir","artist":"Led Zeppelin","title":"Kashmir","album":"Essentials (2020)","genre":"Rock","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/4/art/5b323e7d8298dbe8463fd82c-1660893421.jpg","custom_fields":[]},"elapsed":430,"remaining":87},"playing_next":{"cued_at":1666348028,"played_at":1666348542,"duration":220,"playlist":"TraktaTrax","is_request":false,"song":{"id":"02da9d3fe72b2160c56d1630ec4c1796","text":"Twisted Sister - We're Not Gonna Take It","artist":"Twisted Sister","title":"We're Not Gonna Take It","album":"The Platinum Collection","genre":"","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/4/art/8db10784afa02a546ac8fc54-1660893417.jpg","custom_fields":[]}},"song_history":[{"sh_id":76101,"played_at":1666347688,"duration":346,"playlist":"TraktaTrax","streamer":"","is_request":false,"song":{"id":"4921dc6be0b73196e7cfd1e5df0bee9d","text":"Def Leppard - Love Bites","artist":"Def Leppard","title":"Love Bites","album":"Vault: Def Leppard Greatest Hits","genre":"Metal","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/4/art/64b676d20085c0aa57b98218-1660893420.jpg","custom_fields":[]}},{"sh_id":76099,"played_at":1666347480,"duration":212,"playlist":"TraktaTrax","streamer":"","is_request":false,"song":{"id":"a50b7fbd5e96da42103f929a14891a43","text":"Led Zeppelin - Going To California","artist":"Led Zeppelin","title":"Going To California","album":"Essentials (2020)","genre":"Rock","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/4/art/df9e88deda2dad5f2b4752cc-1660893425.jpg","custom_fields":[]}},{"sh_id":76097,"played_at":1666347204,"duration":278,"playlist":"TraktaTrax","streamer":"","is_request":false,"song":{"id":"0027482e847c081df5563ca2e75ea31d","text":"Led Zeppelin - Misty Mountain Hop","artist":"Led Zeppelin","title":"Misty Mountain Hop","album":"Essentials (2020)","genre":"Rock","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/4/art/067562403fc231320536715c-1660893429.jpg","custom_fields":[]}},{"sh_id":76095,"played_at":1666346942,"duration":264,"playlist":"TraktaTrax","streamer":"","is_request":false,"song":{"id":"d88338016425323179dc819f7a7b7597","text":"Gojira - Another World","artist":"Gojira","title":"Another World","album":"Fortitude","genre":"Rock","isrc":"NLA322000226","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/4/art/7234387227ac0c57e552e19b-1660893419.jpg","custom_fields":[]}},{"sh_id":76092,"played_at":1666346462,"duration":482,"playlist":"TraktaTrax","streamer":"","is_request":false,"song":{"id":"32e56b9923d08f98ba7c0f46ce091aeb","text":"Led Zeppelin - Stairway To Heaven","artist":"Led Zeppelin","title":"Stairway To Heaven","album":"Essentials (2020)","genre":"Rock","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/4/art/c598724c33cad9b2f83057db-1660893420.jpg","custom_fields":[]}}],"is_online":true,"cache":null},{"station":{"id":6,"name":"EVR","shortcode":"evr","description":"eclipse virtual radio","frontend":"icecast","backend":"liquidsoap","listen_url":"https://eclipse-radio.eclipse-streaming.co.za/listen/evr/radio.mp3","url":"https://eclipse-streaming.co.za/radio-player-test/","public_player_url":"https://eclipse-radio.eclipse-streaming.co.za/public/evr","playlist_pls_url":"https://eclipse-radio.eclipse-streaming.co.za/public/evr/playlist.pls","playlist_m3u_url":"https://eclipse-radio.eclipse-streaming.co.za/public/evr/playlist.m3u","is_public":true,"mounts":[{"id":7,"name":"/radio.mp3 (128kbps MP3)","url":"https://eclipse-radio.eclipse-streaming.co.za/listen/evr/radio.mp3","bitrate":128,"format":"mp3","listeners":{"total":0,"unique":0,"current":0},"path":"/radio.mp3","is_default":true}],"remotes":[],"hls_enabled":true,"hls_url":"https://eclipse-radio.eclipse-streaming.co.za/hls/evr/live.m3u8","hls_listeners":0},"listeners":{"total":0,"unique":0,"current":0},"live":{"is_live":false,"streamer_name":"","broadcast_start":null,"art":null},"now_playing":{"sh_id":76108,"played_at":1666348371,"duration":2584,"playlist":"techno","streamer":"","is_request":false,"song":{"id":"5f6346c9a4640827aaab0c0b7b7b1051","text":" - take a chance radio show darryl jooste and michael mayer","artist":"","title":"take a chance radio show darryl jooste and michael mayer","album":"","genre":"","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/6/art/19e55b8a07e51a53000d9874-1663830563.jpg","custom_fields":[]},"elapsed":92,"remaining":2492},"playing_next":null,"song_history":[{"sh_id":76107,"played_at":1666348204,"duration":170,"playlist":"Jazz","streamer":"","is_request":false,"song":{"id":"8b365b4601b53c05bfd19fbf86f99ac7","text":"Leon Berry - Indiana (1)","artist":"Leon Berry","title":"Indiana (1)","album":"Leon Berry Vol.1 (1937-38)","genre":"Jazz","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/6/art/2f9bd99c553bed964704bce8-1663830484.jpg","custom_fields":[]}},{"sh_id":76103,"played_at":1666347832,"duration":375,"playlist":"techno","streamer":"","is_request":false,"song":{"id":"22e89b5a57f95ffedfc9cf687663ecee","text":"Monty Picon - Roberts","artist":"Monty Picon","title":"Roberts","album":"","genre":"","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/6/art/7d639269b7468ab1499defb3-1663830542.jpg","custom_fields":[]}},{"sh_id":76100,"played_at":1666347663,"duration":170,"playlist":"Jazz","streamer":"","is_request":false,"song":{"id":"5fb6e0a23c0fa29add0db555b399d0b0","text":"Leon Berry - Annie Laurie","artist":"Leon Berry","title":"Annie Laurie","album":"Leon Berry Vol.1 (1937-38)","genre":"Jazz","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/6/art/68682e3cb42bcb88ed1478c6-1663830502.jpg","custom_fields":[]}},{"sh_id":76091,"played_at":1666346164,"duration":1501,"playlist":"techno","streamer":"","is_request":false,"song":{"id":"fcdd06271dd4836f3166b2b3034b23f5","text":"take a chance 2 - michael mayer and darryl jooste (1973)","artist":"take a chance 2","title":"michael mayer and darryl jooste (1973)","album":"","genre":"","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/6/art/7f5b69ccf9987652bcd483bc-1663830552.jpg","custom_fields":[]}},{"sh_id":76087,"played_at":1666345925,"duration":242,"playlist":"techno","streamer":"","is_request":false,"song":{"id":"0424871a2309dd3c3ffc5341d7705816","text":"John Roberts - Lesser","artist":"John Roberts","title":"Lesser","album":"","genre":"","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/6/art/1e66c54727b9447efdc8dbfc-1663830544.jpg","custom_fields":[]}}],"is_online":true,"cache":null},{"station":{"id":7,"name":"Radio-Zoibi","shortcode":"radio_zoibi","description":"test for zoibi","frontend":"icecast","backend":"liquidsoap","listen_url":"https://eclipse-radio.eclipse-streaming.co.za/listen/radio_zoibi/jacek","url":"","public_player_url":"https://eclipse-radio.eclipse-streaming.co.za/public/radio_zoibi","playlist_pls_url":"https://eclipse-radio.eclipse-streaming.co.za/public/radio_zoibi/playlist.pls","playlist_m3u_url":"https://eclipse-radio.eclipse-streaming.co.za/public/radio_zoibi/playlist.m3u","is_public":true,"mounts":[{"id":12,"name":"jacek","url":"https://eclipse-radio.eclipse-streaming.co.za/listen/radio_zoibi/jacek","bitrate":128,"format":"mp3","listeners":{"total":0,"unique":0,"current":0},"path":"/jacek","is_default":true},{"id":13,"name":"marekm","url":"https://eclipse-radio.eclipse-streaming.co.za/listen/radio_zoibi/marekm","bitrate":128,"format":"mp3","listeners":{"total":0,"unique":0,"current":0},"path":"/marekm","is_default":true}],"remotes":[{"id":3,"name":"vetsak2 relay","url":"http://vetsak2.antfarm.co.za:8000/azuracast","bitrate":128,"format":"mp3","listeners":{"total":0,"unique":0,"current":0}},{"id":4,"name":"test marek","url":"https://iceant.eclipse-streaming.co.za/azuracast","bitrate":128,"format":"mp3","listeners":{"total":0,"unique":0,"current":0}},{"id":5,"name":"Eclipse radio CDN","url":"http://iceant.antfarm.co.za:8000/azuracast","bitrate":0,"format":"","listeners":{"total":0,"unique":0,"current":0}}],"hls_enabled":true,"hls_url":"https://eclipse-radio.eclipse-streaming.co.za/hls/radio_zoibi/live.m3u8","hls_listeners":0},"listeners":{"total":0,"unique":0,"current":0},"live":{"is_live":false,"streamer_name":"","broadcast_start":null,"art":null},"now_playing":{"sh_id":76109,"played_at":1666348375,"duration":499,"playlist":"zoibi02","streamer":"","is_request":false,"song":{"id":"375f31b90ae67f850566e83fcb5acd84","text":"Antix - Under The Shade","artist":"Antix","title":"Under The Shade","album":"Cavalier","genre":"Electronic","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/7/art/e31a68a92c23168572d8bb4b-1664186214.jpg","custom_fields":[]},"elapsed":88,"remaining":411},"playing_next":null,"song_history":[{"sh_id":76102,"played_at":1666347773,"duration":597,"playlist":"zoibi02","streamer":"","is_request":false,"song":{"id":"499bb93e63c94e367a5509d5c95d5dbe","text":"Antix - Lost & Found (Jamie Stevens Golden Return Remix)","artist":"Antix","title":"Lost & Found (Jamie Stevens Golden Return Remix)","album":"Cavalier","genre":"Electronic","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/7/art/f57f046d5c20fad8a44d8982-1664186399.jpg","custom_fields":[]}},{"sh_id":76098,"played_at":1666347479,"duration":300,"playlist":"zoibi02","streamer":"","is_request":false,"song":{"id":"84645c782aa8db68385fb64d3e2db2dc","text":"Benji Vaughan - Catherine Wheel","artist":"Benji Vaughan","title":"Catherine Wheel","album":"Even Tundra","genre":"Electronic","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/7/art/c51f65ea2b7ae11f6d00f276-1664193283.jpg","custom_fields":[]}},{"sh_id":76096,"played_at":1666347044,"duration":439,"playlist":"zoibi02","streamer":"","is_request":false,"song":{"id":"c77f191e2057e7811063d8ada87e2a7d","text":"Antix - The Hoard (Tim Richards Remix)","artist":"Antix","title":"The Hoard (Tim Richards Remix)","album":"Cavalier","genre":"Electronic","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/7/art/57e2b07fb326b806b0c60839-1664186564.jpg","custom_fields":[]}},{"sh_id":76094,"played_at":1666346779,"duration":263,"playlist":"zoibi02","streamer":"","is_request":false,"song":{"id":"e26a6cda70686355a7f401bca00d85dd","text":"One Mighty Atom - Androids and Angels","artist":"One Mighty Atom","title":"Androids and Angels","album":"Mystery Karaoke","genre":"","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/7/art/eb010d820a353dabe44b1e63-1664184011.jpg","custom_fields":[]}},{"sh_id":76093,"played_at":1666346569,"duration":217,"playlist":"zoibi02","streamer":"","is_request":false,"song":{"id":"7a27151152fdd661b9c775926e36f044","text":"Porno for Pyros - Pets","artist":"Porno for Pyros","title":"Pets","album":"Porno for Pyros","genre":"Alternative","isrc":"","lyrics":"","art":"https://eclipse-radio.eclipse-streaming.co.za/api/station/7/art/308db2f1ba63f90a3bb44a49-1664186950.jpg","custom_fields":[]}}],"is_online":true,"cache":null}]

**Comment by BusterNeece** [comment]
@marekdziem Found and fixed an issue with the PHP API client. Try updating and installing again.

**Comment by maaakmal** [comment]
Shotto thanks Buster
Marek trying update and re-install

**Comment by marekdziem** [comment]
@maaakmal  @BusterNeece 
thanks for your help...
but still having the same issue!!

can we confirm that AZURACAST_BASE_URL is the direct url to the root of the azurcast instance - ie: 
http://172.25.26.153
or 
https://eclipse-radio.eclipse-streaming.co.za

without trailing / 
??

**Comment by BusterNeece** [comment]
@marekdziem Either should work. In my tests with your base URL, it appears you may have some sort of rate limiting taking place at your host level, because my initial request was successful, and then all subsequent ones timed out.

In my local testing, it seems to still be using a timeout of 5 seconds instead of 15, which I'll look into later.

**Comment by marekdziem** [comment]
> @marekdziem Found and fixed an issue with the PHP API client. Try updating and installing again.

@BusterNeece can you confirm our azuracast instance returns the curl request to /api/nowplaying  correctly?
https://eclipse-radio.eclipse-streaming.co.za/api/nowplaying
request to ip:
https://165.227.140.167/api/nowplaying

**Comment by marekdziem** [comment]
> 

oh that's weird!  instance is hosted on standard digital ocean droplet!  is there any settings there to change that could affect this?

**Comment by adoboligh1** [comment]
Stop exposing your ip, do you want to be attacked?

**Comment by marekdziem** [comment]
ok sorry! :)
erm...why? ...are you gonna attack! hehe ;

**Comment by adoboligh1** [comment]
I'm not, just not advisable to share your hetzner dedicated server instance
ip. Lol

**Comment by BusterNeece** [comment]
@marekdziem I pushed an update to AzuraRelay that will ensure it properly has the config handed over to the API client. Update and try again if you don't mind!

**Comment by BusterNeece** [comment]
@marekdziem See above; I'd like to know if this issue has been fixed by the latest update, please try it and let me know.

**Comment by IngeEclipse** [comment]
Thanks for the help and support Buster


From: Buster Neece ***@***.***>
Reply-To: AzuraCast/AzuraCast ***@***.***>
Date: Sunday, 20 November 2022 at 02:57
To: AzuraCast/AzuraCast ***@***.***>
Cc: Inge Dziembowski ***@***.***>, Comment ***@***.***>
Subject: Re: [AzuraCast/AzuraCast] cannot expose AZURACAST_BASE_URL to azurarelay (Issue #5793)


Closed #5793<https://github.com/AzuraCast/AzuraCast/issues/5793> as completed.
